pkg/provisioningv2/capi/capi.go

```diff
@@ -13,10 +13,15 @@ import (
 	"k8s.io/client-go/tools/record"
 	clusterv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	clusterv1alpha4 "sigs.k8s.io/cluster-api/api/v1alpha4"
+	clusterv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/cluster-api/controllers"
 	"sigs.k8s.io/cluster-api/controllers/remote"
 	addonsv1alpha3 "sigs.k8s.io/cluster-api/exp/addons/api/v1alpha3"
-	expv1alpha3 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
+	addonsv1alpha4 "sigs.k8s.io/cluster-api/exp/addons/api/v1alpha4"
+	addonsv1beta1 "sigs.k8s.io/cluster-api/exp/addons/api/v1beta1"
+	expvv1alpha3 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
```

Comment:
Should this be `expv1alpha3`? Same question for `alpha4` and `beta1`.

Reply:
Fixed.